system/autoware_error_monitor/config/autoware_error_monitor.param.yaml

```diff
@@ -0,0 +1,26 @@
+## Default parameters for autoware_error_monitor node
+---
+/**:
+  ros__parameters:
+    update_rate: 100000
+    required_conditions:
+      manual_driving: ['vehicle']
+      autonomous_driving: ['control', 
+        'localization', 
+        'map', 
+        'perception',
+        'planning',
+        'safety',
+        'sensing',
+        'vehicle'
+      ]
+      remote_control: ['control',
+        'remote_control',
+        'safety',
+        'vehicle'
+      ]
+      safe_stop: ['map',
+        'emergency_system',
+        'vehicle'
+      ]
+      emergency_stop: ['vehicle']
```

Comment:
Since we set root path for /diagnostic_agg to `autoware` (currently `system`), these key should also be `autoware/vehicle` instead of `vehicle`. Same with other keys.

Reply:
Ok I wasn't aware of this, I will fix this up. Thanks for the information. Done in d38ebef

Reply:
I don't quite understand, sorry. When do we set the root root path of /diagnostics_agg to `autoware`? Do you mean that in the other configuration files for the diagnostic aggregator the `system` namespace should be changed to `autoware`? Or do you mean in this configuration file, the keys `vehicle`, `map`, `control`, etc should be prefixed with the `autoware/`?

Reply:
since we changed the root of diagnostic_aggregator to `autoware` we also have to modify the keys that we specify in his param file accordingly.

Reply:
Done in 0e7c769